Configurable Monitoring Concept Proposal (#685)
kyma-project/community: concepts/observability-configurable-monitoring/README.md

# Dynamic Monitoring Backend Configuration

> This folder contains pages for evaluating and proposing a concept for having the monitoring backends (and general telemetry backends) in a Kyma cluster configurable. It mainly tries to propose a concept for [Configurable monitoring #13079](https://github.com/kyma-project/kyma/issues/13079).

Reviewer: ```suggestion
This folder contains documents for evaluating and proposing a concept to make the monitoring backends (and general telemetry backends) in a Kyma cluster. The documents mainly try to propose a concept for [configurable monitoring #13079](https://github.com/kyma-project/kyma/issues/13079).
```

> For that, the following pages guide you through the different steps of the proposal:

Reviewer: ```suggestion
For that, the following documents guide you through the different steps of the proposal:
```

* [Ground work - current state, requirements, goal](./01-groundwork.md)

> * [Agent Technology Selection](./02-agents.md)

Reviewer: ```suggestion
* [Agents](./02-agents.md) - technology evaluation and selection
```
